napari/_qt/_qapp_model/qactions/_layer.py

```diff
@@ -0,0 +1,187 @@
+        return
+    if loaded is None:
+        show_warning('Cannot access clipboard')
+        return
+
+    for layer in ll.selection:
+        for key in loaded:
+            loaded_attr_value = loaded[key]
+            if isinstance(loaded_attr_value, list):
+                loaded_attr_value = np.array(loaded_attr_value)
+            if key == 'shear':
+                loaded_attr_value = loaded_attr_value[
+                    -(layer.ndim * (layer.ndim - 1)) // 2 :
+                ]
+            elif key == 'affine':
+                loaded_attr_value = loaded_attr_value[
+                    -(layer.ndim + 1) :, -(layer.ndim + 1) :
+                ]
+            elif isinstance(loaded_attr_value, np.ndarray):
+                if loaded_attr_value.ndim == 1:
+                    loaded_attr_value = loaded_attr_value[-layer.ndim :]
+                elif loaded_attr_value.ndim == 2:
+                    loaded_attr_value = loaded_attr_value[
+                        -layer.ndim :, -layer.ndim :
+                    ]
+
+            setattr(layer, key, loaded_attr_value)
+
+
+def is_valid_spatial_in_clipboard() -> bool:
+    try:
+        loaded = _get_spatial_from_clipboard()
+    except (json.JSONDecodeError, pickle.UnpicklingError):
+        return False
+    if not isinstance(loaded, dict):
+        return False
+
+    return set(loaded).issubset(
+        {'affine', 'rotate', 'scale', 'shear', 'translate'}
+    )
+
+
+Q_LAYER_ACTIONS = [
+    Action(
+        id='napari.layer.copy_all_to_clipboard',
+        title=trans._('Copy all to clipboard'),
+        callback=_copy_spatial_to_clipboard,
+        menus=[{'id': MenuId.LAYERS_COPY_SPATIAL}],
+    ),
+    Action(
+        id='napari.layer.copy_affine_to_clipboard',
+        title=trans._('Copy affine to clipboard'),
+        callback=_copy_affine_to_clipboard,
+        menus=[{'id': MenuId.LAYERS_COPY_SPATIAL}],
+    ),
+    Action(
+        id='napari.layer.copy_rotate_to_clipboard',
+        title=trans._('Copy rotate to clipboard'),
+        callback=_copy_rotate_to_clipboard,
+        menus=[{'id': MenuId.LAYERS_COPY_SPATIAL}],
+    ),
+    Action(
+        id='napari.layer.copy_scale_to_clipboard',
+        title=trans._('Copy scale to clipboard'),
+        callback=_copy_scale_to_clipboard,
+        menus=[{'id': MenuId.LAYERS_COPY_SPATIAL}],
+    ),
+    Action(
+        id='napari.layer.copy_shear_to_clipboard',
+        title=trans._('Copy shear to clipboard'),
+        callback=_copy_shear_to_clipboard,
+        menus=[{'id': MenuId.LAYERS_COPY_SPATIAL}],
+    ),
+    Action(
+        id='napari.layer.copy_translate_to_clipboard',
+        title=trans._('Copy translate to clipboard'),
+        callback=_copy_translate_to_clipboard,
+        menus=[{'id': MenuId.LAYERS_COPY_SPATIAL}],
+    ),
+    Action(
+        id='napari.layer.paste_spatial_from_clipboard',
+        title=trans._('Apply scale/transform from Clipboard'),
```

Comment:
```suggestion
        title=trans._('Apply scale/transforms from Clipboard'),
```
Let's make it plural